src/test/java/uk/gov/justice/hmpps/prison/repository/CaseNoteRepositoryTest.java

```diff
@@ -53,7 +53,7 @@ public void testGetCaseNoteTypesByCaseLoadType() {
 
     @Test
     public void testGetCaseNoteTypeWithSubTypesByCaseLoadType() {
-        final var types = repository.getCaseNoteTypesWithSubTypesByCaseLoadType("COMM");
+        final var types = repository.getCaseNoteTypesWithSubTypesByCaseLoadType("COMM", true);
```

Comment:
'true' isn't expressive here, I have to drill into code to find out what effect 'true' has